decide/voting/tests.py

```diff
@@ -591,3 +618,57 @@ def createCensusEmptyError(self):
             self.cleaner.current_url
             == self.live_server_url + "/admin/voting/question/add/"
         )
+
+
+class BooleanQuestionTestCase(BaseTestCase):
+    def setUp(self):
+        self.question = Question.objects.create(
+            question_type="B", desc="What is your favorite color?", voteBlank=False
+        )
+        self.question.save()
+
+    def test_save_boolean_question(self):
+        self.assertEqual(QuestionOption.objects.count(), 2)
+        self.assertEqual(QuestionOption.objects.first().option, "Sí")
+        self.assertEqual(QuestionOption.objects.last().option, "No")
+
+    def test_str_representation(self):
+        self.assertEqual(str(self.question), "What is your favorite color?")
+
+    def test_one_option_boolean_question(self):
+        self.question = Question.objects.create(
+            question_type="B", desc="What is your favorite color?", voteBlank=False
+        )
+        self.question.save()
+
+        opt = QuestionOption(question=self.question, option="option {}".format(1))
+        with self.assertRaises(BadRequest):
+            opt.save()
+
+    def test_multiple_option_boolean_question(self):
+        self.question = Question.objects.create(
+            question_type="B", desc="What is your favorite color?", voteBlank=False
+        )
+        self.question.save()
+
+        opt1 = QuestionOption(question=self.question, option="option {}".format(1))
+        opt2 = QuestionOption(question=self.question, option="option {}".format(2))
+        opt3 = QuestionOption(question=self.question, option="option {}".format(3))
+        with self.assertRaises(BadRequest):
+            opt1.save()
+            opt2.save()
+            opt3.save()
```

Comment:
Again, I don't undertstand what you are doing here

Reply:
Same case as before